app/src/main/res/values/strings.xml

```diff
@@ -2,7 +2,13 @@
 <resources>
 
     <string name="app_name">XO Robot Hash</string>
-    <string name="hello_world">Hello world!</string>
     <string name="action_settings">Settings</string>
+    <string name="robot_text_message">My XO\'s Robot is</string>
+    <string name="robot_image_internal_filename">this-xos-robot2.png</string>
```

Comment:
It's not necessary to put this one in `strings.xml`, just use a static final in the activity.